app.rb

```diff
@@ -0,0 +1,147 @@
+require_relative './book'
+require_relative './teacher'
+require_relative './student'
+require_relative './person'
+require_relative './rental'
+# App object class
+class App
+  def initialize
+    @book_list = []
+    @rentals = []
+    @person = []
+  end
+
+  def menu
+    puts "Welcome to school library ! \n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
+    gets.chomp
+  end
```

Comment:
- [ ] <b>REQUIRED:</b> You still haven't fixed some issues raised from the previous review.
  - When I try to create a person with type teacher, the app throws an error:
    ![Screenshot(15)](https://user-images.githubusercontent.com/45217364/177123761-03f481c0-155c-4bf3-ae3f-897aa4c13b79.png)
  - When I try to create a person with student type, the app can't be able to set a permission. 
  ![Screenshot(16)](https://user-images.githubusercontent.com/45217364/177124454-548462e3-2bfc-4ee6-8720-493742e7e07d.png)